cli/src/command.rs

```diff
@@ -276,6 +276,31 @@ pub fn run() -> Result<()> {
 			})?)
 		},
 		Some(Subcommand::Key(cmd)) => Ok(cmd.run(&cli)?),
+		#[cfg(feature = "try-runtime")]
+		Some(Subcommand::TryRuntime(cmd)) => {
+			let runner = cli.create_runner(cmd)?;
+			let chain_spec = &runner.config().chain_spec;
+			set_default_ss58_version(chain_spec);
+
+			runner.async_run(|config| {
+				use sc_service::TaskManager;
+				let registry = config.prometheus_config.as_ref().map(|cfg| &cfg.registry);
+				let task_manager = TaskManager::new(
+					config.task_executor.clone(),
+					registry,
+				).unwrap();
+
+				Ok((
+					cmd.run::<
+						service::kusama_runtime::Block,
+						service::KusamaExecutor,
```

Comment:
The correct way for this would be an if that checks which version we need to call.

Reply:
Hmm strangely then the two features will have incompatible return types (tries coercing both `Ok` and `Err` with `map_ok(|_| ())` and `map_err(|_| ())`). Will play a bit more with it later today, might make an issue for it and leave for some other time.